config.sample.json

```diff
@@ -0,0 +1,56 @@
+{
+    "BCH": {
+        "address": "",
+        "uriScheme": "bitcoincash:",
+        "amountParam": "?amount=",
```

Comment:
Not needed. Instead we'll have a special case for XLM. (Just the `?`). `paramDelimiter` instead of amountParam.